### ISSUE TITLE
Don't work for different dataset (custom data)?

### ISSUE THREAD

**Issue by AI-Hunter** [yanx27/Pointnet_Pointnet2_pytorch#154]
When I use S3DIS dataset, it is working, but when I am using my own dataset, it is stuck at classifier.train()! What is the reason?
 is it normalization issue?

I converted into .npy from my las file using the collect_indoor3d_data.py ( a little modified base on my class number), and I don't have rgb values, so I have set the rgb values with 255.

Is there any suggestion please?

**Comment by gang3217** [comment]
> Have you solved this problem yet

**Comment by wyz5008** [comment]
你好，我也在尝试做自己的语义分割数据集，（已经采集做好了PCD格式的点云文件），请问你是如何打语义分割的标签，并且调整格式以及修改代码使其可以训练呢？

**Comment by bhagatdas** [comment]
Hi... @AI-Hunter  @gang3217 @wyz5008 @lly007 @yushiangw  anyone solved the problem for custom dataset.. I am also stuct at 
BN momentum updated to: 0.100000
  0% 0/6 [00:00<?, ?it/s]
or classifier = classifier.train()
please help

**Comment by sean880304** [comment]
@bhagatdas 
https://github.com/yanx27/Pointnet_Pointnet2_pytorch/blob/eb64fe0b4c24055559cea26299cb485dcb43d8dd/data_utils/S3DISDataLoader.py#L59
change this number to a smaller one (say 120), this may resolve your problem.

**Comment by bhagatdas** [comment]
@sean880304  Yeah.. Did the same while ago... now it is working well with the custom dataset.

**Comment by foysalahmed10** [comment]
@sean880304 @bhagatdas First, I would like to thank you for your time and support. I'm trying to train my own dataset. Can you tell me how can I prepare my dataset for training, step by step. I already completed labeling, now the dataset in .PCD  and .txt format.

**Comment by bhagatdas** [comment]
I used the S3DIS format to train my own data. So, first go through the S3DIS data format, run the mode using it. after that you ll know how to convert your own dataset to use the model.

**Comment by foysalahmed10** [comment]
@bhagatdas Thank you so much for your reply. I will follow your advice.

**Comment by Akshatha-Jagadish** [comment]
@bhagatdas could you please tell why the point cloud data is not normalized in s3dis data loader? Thank you

**Comment by bhagatdas** [comment]
@Akshatha-Jagadish I don't have any particular answer, but if you are doing it, let me know the result. I was getting around 88% accuracy with Toronto-3D dataset.
